doc/tutorials/izhikevich/nestml_izhikevich_tutorial.ipynb

```diff
@@ -193,13 +185,32 @@
     "fig.show()"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Task 2: Parameter space exploration\n",
+    "-----------------------------------\n",
+    "\n",
+    "Perform a parameter space exploration to reproduce the bottom eight panels from [1], figure 2."
+   ]
+  },
+  {
+   "cell_type": "code",
```

Comment:
Is this code block really required?

Reply:
Not strictly speaking, but it helps to show where you are expected to write some code. If you feel it's better without, I will remove it.

Reply:
It looks somewhat like an incomplete code block to me, even though that is the motive of having this code block. Maybe you could add a comment something like: `# Write your code here`?

Reply:
Code block has been removed :)

Reply:
Thanks, sorry to be nitpicky about this!